src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  text-transform: uppercase;
```

Comment:
it's a bad idea. For this task, it's work correctly, but in other sites (with more content) all text will be in uppercase